pkgs/development/ocaml-modules/sedlex/default.nix

```diff
@@ -11,12 +11,9 @@
 
 let param =
   if lib.versionAtLeast ppxlib.version "0.26.0" then
-    if lib.versionAtLeast ocaml.version "4.14" then {
-      version = "3.1";
-      sha256 = "sha256-qG8Wxd/ATwoogeKJDyt5gkGhP5Wvc0j0mMqcoVDkeq4=";
-    } else {
-      version = "3.0";
-      sha256 = "sha256-+4ggynMznVfjviMBjXil8CXdMByq4kSmDz6P2PyEETA=";
+    {
+      version = "3.2";
+      sha256 = "sha256-wu4PcMuTBdUWOzKwcls8kEEOVRPnJi7IcKEbAvM5t6E=";
```

Comment:
Is this hash correct?

Reply:
Hm, looks like there might be something weird going on with the release; the release-mail I got from GitHub says the tag points at https://github.com/ocaml-community/sedlex/commit/d126b58b3cf8acdb03a89f85e3db27e140526c8f, but it looks like it's been updated to https://github.com/ocaml-community/sedlex/commit/a1362bd69aec6877cb9f3b950c7f4e591e2d69ee since? I'll update it.